package.json

```diff
@@ -80,6 +80,7 @@
     "@storybook/node-logger": "^6.1.21",
     "@storybook/preset-create-react-app": "^3.1.7",
     "@storybook/react": "^6.1.21",
+    "babel-eslint": "^10.1.0",
```

Comment:
Why this addition channel, Do we really neeed `babel-eslint` parser.

Reply:
Yeah initially I needed it before I added test to gitignore.. while reverting the changes I guess this got reverted as well.. I will make sure I remove this

Reply:
@Abhishek-kumar09 removed the babel-eslint from both package.Json and config

Reply:
@Abhishek-kumar09 made the new changes u asked

Reply:
@Abhishek-kumar09 please do check the final PR or if any changes are there then more merge conflicts can occur with new pushes to the repo